cfme/configure/access_control.py

```diff
@@ -1213,6 +1218,7 @@ def is_displayed(self):
 class AllTenantView(ConfigurationView):
     """ All Tenants View """
     toolbar = View.nested(AccessControlToolbar)
+    table = ListEntity
```

Comment:
**REQURIED**
It shouldn't be named as table, but `list_entity`.

Reply:
Fixed